golang/x/relay/alias.go

```diff
@@ -27,6 +27,8 @@ var (
 	NewMsgMarkNewHeaviest = types.NewMsgMarkNewHeaviest
 	// NewMsgGetRequest is what is says on the tin
 	NewMsgNewRequest = types.NewMsgNewRequest
+	// NewMsgProvideProof is what is says on the tin
+	NewMsgProvideProof = types.NewMsgProvideProof
```

Comment:
I have no idea why this is here :)

Reply:
basically the `alias` file gives a single location where other module-writers can get your types and datastructures from.
so that other modules can use `relay` instead of `relay` and `relay/types` and `btcspv` and `btcspv/types`

Reply:
makes sense!